app/app/src/main/java/com/example/clubwat/views/AddEventView.kt

```diff
@@ -0,0 +1,220 @@
+                    Text(
+                        text = "Add Event",
+                        fontWeight = FontWeight.Bold,
+                        fontSize = 20.sp
+                    )
+                },
+                backgroundColor = LightYellow,
+                contentColor = Color.Black
+            )
+        },
+        content = {
+            Column(
+                modifier = Modifier
+                    .padding(it)
+                    .fillMaxSize()
+                    .padding(14.dp),
+                verticalArrangement = Arrangement.spacedBy(10.dp)
+            ) {
+                if (clubId == null) return@Scaffold
+                Column() {
+                    Spacer(modifier = Modifier.height(5.dp))
+                    Text(text = "Title", style = TextStyle(fontWeight = FontWeight.Bold))
+                    OutlinedTextField(
+                        value = viewModel.title.value,
+                        onValueChange = { newValue: String -> viewModel.title.value = newValue },
+                        modifier = Modifier
+                            .padding(vertical = 8.dp)
+                            .fillMaxWidth(),
+                        colors = TextFieldDefaults.colors(
+                            focusedIndicatorColor = Orange,
+                            unfocusedContainerColor = Color.Transparent,
+                            focusedContainerColor = Color.Transparent
+                        ),
+                        singleLine = true
+                    )
+                }
+
+                Column() {
+                    Text(text = "Description", style = TextStyle(fontWeight = FontWeight.Bold))
+                    OutlinedTextField(
+                        value = viewModel.description.value,
+                        onValueChange = { newValue: String -> viewModel.description.value = newValue },
+                        modifier = Modifier
+                            .padding(vertical = 8.dp)
+                            .fillMaxWidth(),
+                        colors = TextFieldDefaults.colors(
+                            focusedIndicatorColor = Orange,
+                            unfocusedContainerColor = Color.Transparent,
+                            focusedContainerColor = Color.Transparent
+                        ),
+                        singleLine = true
+                    )
+                }
+
+                Column() {
+                    Text(text = "Start Date", style = TextStyle(fontWeight = FontWeight.Bold))
+                    DateTimePickerView(dateTime = viewModel.startDateTime)
+                }
+
+                Column() {
+                    Text(text = "End Date", style = TextStyle(fontWeight = FontWeight.Bold))
+                    DateTimePickerView(dateTime = viewModel.endDateTime)
+                }
+
+                Column() {
+                    Text(text = "Location", style = TextStyle(fontWeight = FontWeight.Bold))
```

Comment:
This should be a text field not a radio button

Reply:
resolved.